include/ajax.tickets.php

```diff
@@ -64,6 +64,7 @@ function lookup() {
             return $this->lookup();
         }
 
+        // TODO: Why not aggregate based on relationship?
```

Comment:
@aydreeihn - please review this commit. Basic optimization due to a warning I saw. We most definitely need to revisit merge feature at some point.